docs/content/preview/explore/transactions/distributed-transactions-ysql.md

```diff
@@ -201,7 +209,7 @@ where transaction_mode is one of:
   [ NOT ] DEFERRABLE
 ```
 
-### `transaction_mode`
+### transaction_mode
 
 The `transaction_mode` can be set to one of the following options:
 
```

Comment:
_Users can perform read or write operations._ and _Users can perform read operations only._   Who are these users? Shouldn't it be "you"?

_As an example, trying to do a write operation such as creating a table or inserting a row in a READ ONLY transaction would result in an error as shown below._ -->  As an example, trying to do a write operation such as creating a table or inserting a row in a READ ONLY transaction would result in an error: